src/codegen.cpp

```diff
@@ -5964,7 +5961,7 @@ static void init_julia_llvm_env(Module *m)
         &pow,
```

Comment:
More precisely what `glibc` + `libstdc++`/`libc++` do when `<math.h>` is directly included by c++ code is roughly

```c++
extern double pow(double, double);

namespace std {
using ::pow;
float pow(float, float);
long double pow(long double, long double);
// template version for promotion...
}

using std::pow;
```

It shouldn't be possible to overwrite a previously defined `::pow(double, double)` so I expect MSVC to do something similar and it should be safe in general to specify the overload type using a `static_cast`.